src/components/qas/questions/qaCalculator/qaCalculatorsData.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { QACalculatorIcon, QACalculatorIconUnit } from './qaCalculatorIcons';
+import FindImpactCrater from './equations/FindImpactCrater';
+import FindDiameter from './equations/FindDiameter';
+import FindKineticEnergy from './equations/FindKineticEnergy';
+import {
+  getCalculatedMeasurementsForDistance,
+  calculateDiameter,
+  calculateKineticEnergy,
+  calculateCraterDiameter,
+} from './qaCalculatorUtilities';
+import { col50 } from './qaCalculator.module.scss';
+import Unit from '../../../charts/shared/unit';
+
+export const DistanceCalculator = {
+  value: {
+    magnitude: null,
+    distanceModulus: null,
+    parsecs: null,
+    megaParsecs: null,
+    lightYears: null,
+    megaLightYears: null,
+  },
+  formula: getCalculatedMeasurementsForDistance,
+  inputs: [
+    {
+      containerWidth: col50,
+      leftIcon: <QACalculatorIcon content="m =" />,
+      placeholder: 'magnitude',
+      defaultValue: 'magnitude',
+    },
+  ],
+};
+
+export const KineticEnergyCalculator = {
+  value: {
+    mass: null,
+    velocity: null,
+    kineticEnergy: null,
+  },
+  solutionVariable: 'kineticEnergy',
+  equation: FindKineticEnergy,
+  formula: calculateKineticEnergy,
+  inputs: [
+    {
+      containerWidth: col50,
+      leftIcon: <QACalculatorIcon content="m =" />,
+      rightIcon: <QACalculatorIconUnit unit={<Unit type="mass" />} />,
+      placeholder: 'mass',
+      defaultValue: 'mass',
+    },
+    {
+      containerWidth: col50,
+      leftIcon: <QACalculatorIcon content="v =" />,
+      rightIcon: <QACalculatorIconUnit unit={<Unit type="velocity" />} />,
+      placeholder: 'velocity',
+      defaultValue: 'velocity',
+    },
+  ],
+};
+
+export const SizeCalculator = {
+  value: {
+    magnitude: null,
+    albedo: null,
+    diameter: null,
+  },
+  solutionVariable: 'diameter',
+  equation: FindDiameter,
+  formula: calculateDiameter,
+  inputs: [
+    {
+      containerWidth: col50,
+      leftIcon: <QACalculatorIcon content="H =" />,
+      placeholder: 'magnitude',
+      defaultValue: 'magnitude',
+    },
+    {
+      containerWidth: col50,
+      leftIcon: <QACalculatorIcon content="p =" />,
+      placeholder: 'albedo',
+      defaultValue: 'albedo',
+    },
+  ],
+};
+
+export const ImpactCalculator = {
+  value: {
+    craterDiameter: null,
+    density: null,
+    velocity: null,
+    asteroidDiameter: 156,
```

Comment:
typo?

Reply:
yeah, it was apart of my testing...forgot about it. It has been updated now. 👍